api/src/test/java/org/openmrs/module/appointmentscheduling/api/AppointmentTypeServiceTest.java

```diff
@@ -152,7 +152,7 @@ public void saveAppointmentType_shouldSaveNewAppointmentType() throws Exception
 		List<AppointmentType> appointmentTypes = service.getAppointmentTypes("Some Name");
 		assertEquals(0, appointmentTypes.size());
 		
-		AppointmentType appointmentType = new AppointmentType("Some Name", "Description", 10);
+		AppointmentType appointmentType = new AppointmentType("Some Name", "Description", 10, null);

```

Comment:
See my comment above... in the end, this class shouldn't need to change because we wouldn't be changing the existing AppointmentType constructor, but rather creating a new, additional constructor.